apps/super-app/src/index.tsx

```diff
@@ -3,14 +3,15 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { BrowserRouter as Router } from 'react-router-dom';
 
 const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement
 );
 root.render(
-  <React.StrictMode>
+  <Router>
     <App />
-  </React.StrictMode>
+  </Router>
```

Comment:
Why does this remove `<React.StrictMode>`? I would expect `<Router>` to be inside `<React.StrictMode>`, not that we would remove it altogether.